api/CSSTransformComponent.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "CSSTransformComponent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSTransformComponent",
+        "support": {
+          "chrome": {
+            "version_added": "66"
+          },
+          "chrome_android": {
+            "version_added": "66"
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "53"
+          },
+          "opera_android": {
+            "version_added": "53"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": "66"
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "stringifier": {
```

Comment:
There isn't actually a property called `stringifier` here, right? I think this is WebIDL markup for saying that there is a `toString` method. https://drafts.css-houdini.org/css-typed-om/#csstransformcomponent

Reply:
In this case it means that the instance variable may be used in contexts where a string is expected. [Go here](https://heycam.github.io/webidl/#idl-stringifiers) and scroll down to example 17.

How do you want to handle this?

Reply:
Thanks for the pointer. I'm not sure about this. I haven't personally documented such a `stringifier` on MDN. Should this really have an own MDN doc page, or should it be documented inline on the interface page for `CSSTransformComponent`? (in that case I think we wouldn't need a compat entry here either). 
If you think it is worth having an own page about, we probably want to proceed with `stringifier` as a subfeature and leave this as is. 
@chrisdavidmills or @a2sheppy, do you have experience with documenting stringifiers on MDN?

Reply:
Is there a way to have it in the compat data without getting its own page? I agree that it should be documented on the interface, but a browser could conceivably implement the interface in one version and stringifier in the next. 

Since Chrome is the only browser for this interface so far, I'm going to remove stringifier. But I'm leaving the question standing because the implementation scenario I've described could happen.

Reply:
Compat data can exist without a dedicated MDN page, just omit the `mdn_url` property. 

I agree this is a case that could exist. Maybe it would be worth to discuss this on discourse and come up with a standard for how we document stringifiers. Maybe you could lead with an example here, if you're intending to document `CSSTransformComponent` soon. I doubt anyone has a standard way for dealing with stringifiers yet and it's generally good if we have some standards for API docs.

Thanks for removing it for now, though. That will unblock merging this.